src/components/About.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { useCounterStore } from "../stores/counter";
+
+const store = useCounterStore();
+
+defineProps<{ msg: string }>();
+</script>
+
+<template>
+  <h1>{{ msg }}</h1>
+  <h1>Gameing</h1>
```

Comment:
While vue3 supports multiple top level divs in a component. It's a good idea to only have one. 

I'd wrap these in a div of some kind.